syntax = "proto3";

message storeData {
    string filename = 1;
    int32 replica_id = 2;
    int32 chunk_id = 3;
    int32 total_chunks = 4;
}

message getData {
    string filename = 1;
}